cmd/soroban-cli/src/commands/contract/mod.rs

```diff
@@ -81,7 +81,7 @@ pub enum Cmd {
     ///     stellar contract invoke ... -- --help
     Invoke(invoke::Cmd),
 
-    /// Optimize a WASM file
+    /// (Deprecated in favor of `build --optimize` subcommand) Optimize a WASM file
```

Comment:
Suggest a deprecated-use statement that is simpler language.
```suggestion
    /// (Deprecated, use `build --optimize` instead) Optimize a WASM file
```